README.md

```diff
@@ -19,32 +19,19 @@ With the [DTDL extension for Visual Studio Code](https://marketplace.visualstudi
 You could use the command palette to create interface from predefined or customized templates. 
 
 - In Visual Studio Code, select **View > Command Palette** to open the VS Code command palette.
-- In the command palette, enter and run the command **DTDL: Create Interface** 
-- Follow the instruction to assign the interface name and choose a template.  
-- A JSON file will be created in the current folder. The file name is based on the input interface name. 
-- You should replace the **{company}** field in @id with your own company’s name. Note that @id is the path component of the [Digital Twin Model Identifier](https://github.com/Azure/opendigitaltwins-dtdl/blob/master/DTDL/v2/dtdlv2.md#digital-twin-model-identifier-dtmi) (DTMI) and should follow the DTMI rule to uniquely identify the device model.
-- You could extend the interface with new types defined in [DTDL v2](https://github.com/Azure/opendigitaltwins-dtdl/blob/master/DTDL/v2/dtdlv2.md). 
+- In the command palette, enter and run the command **DTDL: Create Interface**. 
+- Follow the instruction to assign the interface name. 
+- If there are multiple templates existing, choose a template you need. Otherwise, the extension will the basic template as default.
+- A JSON file will be created in the current folder. The file name is based on the interface name you assigned. 
+- You should replace the **{company}** field in @id with your own company’s name without braces. Note that @id is the path component of the [Digital Twin Model Identifier](https://github.com/Azure/opendigitaltwins-dtdl/blob/master/DTDL/v2/dtdlv2.md#digital-twin-model-identifier-dtmi) (DTMI) and should follow the DTMI rule to uniquely identify the device model.
```

Comment:
The @id value has been updated after Rido's suggestion, need to update the description here. The current default value of @id is "dtmi:com:example:<interface_name>;1", and user don't need to remove the braces.

Reply:
Rewording this as the comment.